utils/serverStatistic.js

```diff
@@ -0,0 +1,130 @@
+  const playerAchievmentsURL = `${BASE_URL}/account/achievements/?application_id=${application_id}&account_id=${playerIds.join(',')}`;
+
+  try {
+    const playerStats = await getInfo(playerStatsURL);
+    const playerAchievments = await getInfo(playerAchievmentsURL);
+
+    for (const accountId in playerStats) {
+      const stats = playerStats[accountId]?.statistics;
+      const achievements = playerAchievments[accountId]?.achievements;
+      const vehicleStats = await getVehicleStats(accountId);
+
+      if (!stats || !achievements) continue;
+      const regularStats = stats.all;
+      const ratingStats = stats.rating;
+
+      if (JSON.stringify(allPlayerStats.account) === '{}') {
+        allPlayerStats.account.regular = getStatsObject(statsNames, regularStats);
+        allPlayerStats.account.rating = getStatsObject(statsNames, ratingStats);
+        allPlayerStats.account.mastery = getStatsObject(achievementsName, achievements);
+      } else {
+        statsNames.map((statName) => {
+          allPlayerStats.account.regular[statName] = allPlayerStats.account.regular[statName] + regularStats[statName];
+          allPlayerStats.account.rating[statName] = allPlayerStats.account.rating[statName] + ratingStats[statName];
+        });
+        achievementsName.map((achievementName) => {
+          allPlayerStats.account.mastery[achievementName] = allPlayerStats.account.mastery[achievementName] + (achievements[achievementName] || 0);
+        });
+      }
+
+      if (!allPlayerStats.tanks.length && vehicleStats) allPlayerStats.tanks.push(...vehicleStats);
+      else if (vehicleStats) {
+        vehicleStats.map((vehicle) => {
+          const { wotId, regular, battleLifeTime, mastery } = vehicle;
+          if (allPlayerStats.tanks.find((tank) => tank.wotId === wotId)) {
+            const existingTank = allPlayerStats.tanks.find((tank) => tank.wotId === wotId);
+            statsNames.map((statName) => {
+              existingTank.regular[statName] += regular[statName];
+            });
+            achievementsName.map((achievementName) => {
+              existingTank.mastery[achievementName] += mastery[achievementName] || 0;
+            });
+          } else {
+            allPlayerStats.tanks.push({ wotId, regular, battleLifeTime, mastery });
+          }
+        });
+      } 
+    }
+  } catch (error) {
+    console.error('Error:', error);
+  }
+}
+
+const fetchAllPlayerStats = async () => {
+  try {
+    const numRequests = Math.ceil(TOTAL_ACCOUNTS / MAX_ACCOUNTS_PER_REQUEST);
+
+    for (let i = 5948593; i < numRequests; i++) { // change `i` to any number
+      const startIndex = i * MAX_ACCOUNTS_PER_REQUEST;
+      const endIndex = Math.min(startIndex + MAX_ACCOUNTS_PER_REQUEST, TOTAL_ACCOUNTS);
+      const accountIds = [];
+
+      for (let accountId = startIndex; accountId < endIndex; accountId++) {
+        accountIds.push(accountId);
+      }
```

Comment:
Create the accountIds array using Array.from and the calculate startIndex and endIndex. This avoids the need for an extra loop to populate the array.
```suggestion
    const accountIds = Array.from({ length: endIndex - startIndex }, (_, index) => startIndex + index);
```